feature/agenda/data/src/main/java/com/sergius/agenda/data/HttpRemoteEventDataSource.kt

```diff
@@ -13,41 +17,55 @@ import com.sergius.core.domain.util.DataError
 import com.sergius.core.domain.util.EmptyResult
 import com.sergius.core.domain.util.Result
 import com.sergius.core.domain.util.asEmptyDataResult
+import com.sergius.core.domain.util.onSuccess
 import io.ktor.client.HttpClient
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.async
-import kotlinx.coroutines.flow.Flow
 import java.time.LocalDate
 
+private const val _EVENT = "/event"
```

Comment:
You could make this a bit cleaner by centralizing those route strings in a `companion object`. That way you don’t have constants floating around at the top.
 For example
```kotlin

class HttpRemoteEventDataSource(
    private val httpClient: HttpClient,
    private val applicationScope: CoroutineScope,
) : RemoteEventDataSource {

    override suspend fun createEvent(event: Event): EmptyResult<DataError.Network> {
        ...
    }

    companion object {
        private const val EVENT = "/event"
    }
}
```